content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md

```diff
@@ -1,10 +1,201 @@
+UPDATE - Takes(s): 20.0, Count: 15799, OPS: 791.1, Avg(us): 19834, Min(us): 10505, Max(us): 41090, 99th(us): 35000, 99.9th(us): 40000, 99.99th(us): 41000
+...
+```
+
+Normally, this workload above will run several minutes, you will have enough time to check and manipulate the cluster.
+
+## Step 5. Check the workload
+
+1. Go to the [Grafana](https://grafana.com) page at [http://127.0.0.1:3000](http://127.0.0.1:3000)
+
+2. Login with default username `admin` and password `admin`
+
+3. Go to dashboard **playground-tikv-summary**, the OPS information is in panel **gRPC message count** in row **gRPC**.
+
+{{< figure
+    src="/img/docs/check-ops.png"
+    width="80"
+    number="1" >}}
+
+4. By default, TiKV replicates all data 3 times and balances it across all stores. To see this balance, go to page **playground-overview** and check the region count across all nodes. In this example, we load a small amount of data, thus only one region is presented:
+
+{{< figure
+    src="/img/docs/fault-tolerance-region-count.png"
+    width="80"
+    number="1" >}}
+
+## Step 6. Simulate a single node failure
+
+To understand fault tolerance in TiKV, it's important to review a few concepts from the [architecture](https://github.com/tikv/tikv#tikv-software-stack).
+| Concept        |                                                   Description                                                    |
+| -------------- | :--------------------------------------------------------------------------------------------------------------: |
+| **Raft Group** |                  Each replica of a region is called Peer. All of such peers form a raft group.                   |
+| **Leader**     | In every raft group, there is a unique role called leader, are responsible for read/write requests from clients. |
+
+
+Notice that all read/write operations are handled by the leader of the region group, we are going to stop the only one leader in the cluster, and check the load continuity and cluster health.
+
+
+1. Go to Grafana dashboard **playground-overview**, the leader distribution is in panel **leader** in row **TiKV**.
+
+2. In the example, the local process that open port `20180` hold only one leader in the cluster. Use the following command to stop the process.
+    ```sh
+    kill -STOP $(lsof -i:20180 | grep tikv | head -n1 | awk '{print $2}')
+    ```
+
+## Step 7. Check load continuity and cluster health
+
+1. Check the leader distribution again, you will find the leader is moving another store.
+
+{{< figure
+    src="/img/docs/fault-tolerance-leader-recover.png"
+    width="80"
+    number="1" >}}
+
+2. Check the gRPC OPS, you will find there is a small duration that the TiKV is unavailable because the leader was down. However, the workload is back online as soon as the [election](https://raft.github.io/raft.pdf) is completed.
```

Comment:
```suggestion
2. Check the gRPC OPS, you will find there is a small duration that the TiKV is unavailable because the leader was down. However, the workload is back online as soon as the election is completed.
```

Reply:
Because there is no such reference in the docs. I suggest keeping this link due to not all people know about the concept of `election`.